.github/workflows/checks.yml

```diff
@@ -22,11 +22,19 @@ on:
       - ready_for_review
       - reopened
       - synchronize
+    paths-ignore:
+    # Stop quick checks if the PR contains changes limited to just these files
+      - 'CHANGELOG.md'
+      - 'version/VERSION'
   push:
     branches:
       - '*'
     tags:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
+    paths-ignore:
+    # Stop quick checks if the PR contains changes limited to just these files
+      - 'CHANGELOG.md'
+      - 'version/VERSION'
```

Comment:
I think this stops the check running on the commit made when the PR is merged, so this PR will need to be updated with triggers specific to different subsets of branches (paths-ignore for pushes to contributors' branches, no ignoring for pushes to branches like main and vX.X.X)